openhands/llm/fn_call_converter.py

```diff
@@ -319,9 +319,8 @@ def convert_fncall_messages_to_non_fncall_messages(
     converted_messages = []
     first_user_message_encountered = False
     for message in messages:
-        role, content = message['role'], message['content']
-        if content is None:
-            content = ''
+        role = message['role']
+        content = message.get('content', '')
```

Comment:
😓 I was actually just debugging this a few hours ago -- I'll report back if I find anything. It feels to me that it could be LLM serving issue, but i could be wrong.

Reply:
I wonder if [this code](https://github.com/All-Hands-AI/OpenHands/blob/1acb66c2b3dc7435b8518761a8afeb757a774f8d/openhands/core/message.py#L112) made it happen. I don't seem to reproduce it anymore on this branch, because I converted them using the string serializer (for non-native function calling)